commands/abstractCommand.ts

```diff
@@ -2,7 +2,7 @@ import type PeerId from 'peer-id'
 import chalk from 'chalk'
 
 export type AutoCompleteResult = [string[], string] 
-export const emptyAutoCompleteResult = (line: string):AutoCompleteResult => [[''], line]
+export const emptyAutoCompleteResult = (line: string):AutoCompleteResult => [[], line]
```

Comment:
This actually won't work - it needs the empty string to know that the result is empty, rather than all options.

Reply:
how can I reproduce a scenario like that? is this correct `ping foo`?

Reply:
I think (although it's been a week since I looked at this) that the flow to reproduce is:
for ping foo

'Ping Fo' Tab -> shows ping foo
'Ping Fog' Tab -> No results

Reply:
reverted to using `[[''], line]`